client/we_meet_client/src/__features__/homepage_navbar.feature.js

```diff
@@ -8,7 +8,7 @@ describe('Homepage', () => {
         await page.reload();
     });
 
-    describe('navbar', () => {
+    describe('displays correct buttons in navbar', () => {
 
```

Comment:
Can this be worded differently?  How would one display an incorrect button?

Reply:
We just want to make sure the buttons are displayed.